src/client.ts

```diff
@@ -13,12 +13,17 @@ export class Request {
     this.headers[key] = value;
   }
 
-  get(requestUrl: string) {
+  get(
+    requestUrl: string,
+    path: string = "",
```

Comment:
Let's not add path here and handle using string literals for now. Adding `undefined` due to this is code smell. 

We should definitely handle path params too. But we have to figure out another way to do so. I'll create another issue for this.

This will also ensure that we are staying in the scope of this PR

Reply:
Also this method will not be valid if we have multiple path params, eg. 

`/regression/${appId}/testcases/${testCaseId}`

Reply:
you're right. I removed it and pushed the changes.